django_cloud_deploy/crash_handling/__init__.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A module handles crashes of the tool."""
+
+import os
+import platform
+import subprocess
+import sys
+import traceback
+import urllib.parse
+import webbrowser
+
+import jinja2
+
+from django_cloud_deploy.cli import io
+from django_cloud_deploy import __version__
+
+# TODO: Add an issue label for issues reported by users
+_REQUEST_URL_TEMPLATE = (
+    ('https://github.com/GoogleCloudPlatform/django-cloud-deploy/issues/new?'
```

Comment:
Maybe include a title so the user just has to hit submit? I'd suggest:
"{str(Exception)} during "{command}"" e.g.
"TypeError: cannot  during str to int during ""django-cloud-deploy new"

But it doesn't really matter

Reply:
Done. I also added exception type in the title. For example, "{type(err).__name__}: {str(Exception} during "{command}"". If a "KeyError" is raised, then "str(Exception)" is the key. It can be anything. So we might need the exception type.